script/app.js

```diff
@@ -56,6 +70,15 @@ const renderTodos = () => {
     reversedTodos.forEach((task) => {
         const taskCard = getTodoCard(task);
         taskListContainerElement.appendChild(taskCard);
+        if (task.isCompleted) {
+            const completeButton = taskCard.querySelector(
+                ".task-card__icon--complete"
+            );
+            const editButton = taskCard.querySelector(".task-card__icon--edit");
+            task.isCompleted &&
+                (completeButton.style.display = editButton.style.display =
```

Comment:
maybe we can take another classname in both of them, so that with one line you can hide those items.